libr/core/cmd_print.c

```diff
@@ -922,6 +923,28 @@ static void cmd_pdj(RCore *core, const char *arg, ut8* block) {
 	pj_free (pj);
 }
 
+static void cmd_p_minus_e(RCore *core, ut64 at, ut64 ate) {
+	ut8 *blockptr = malloc (ate - at);
+	if (!blockptr) {
+		return;
+	}
+	if (!r_io_read_at (core->io, at, blockptr, (ate - at))) {
+		eprintf ("Error: failed to read block");
+	} else {
+		ut8 entropy = (ut8)(r_hash_entropy_fraction (blockptr, (ate - at)) * 255);
+		entropy = 9 * entropy / 200; // normalize entropy from 0 to 9
+		if (r_config_get_i (core->config, "scr.color")) {
+			const char *color =
+				(entropy > 6) ? Color_BGRED :
+				(entropy > 3) ? Color_BGGREEN :
+				Color_BGBLUE;
+			r_cons_printf (Color_RESET"%s", color);
```

Comment:
nope, thats not going to work. change this line for this:

r_cons_printf ("%s%d"Color_RESET, color, entropy);